org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corrections/proposals/InitializeFinalFieldProposal.java

```diff
@@ -0,0 +1,623 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2020 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copied from /org.eclipse.jdt.ui/src/org/eclipse/jdt/internal/ui/text/correction/proposals/InitializeFinalFieldProposal.java
```

Comment:
@robstryker Can you help @snjeza with this one so there aren't more copied files while you try to remove them?

Reply:
https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/1078/files#diff-8d655d5e2f549b613501debdf5c12178d9cd233db55f3e7c420b06fb4cfec120R36-R39) is not in the latest nightly build - https://download.eclipse.org/eclipse/updates/4.31-I-builds/I20240116-1800
I will update the PR when the next nightly build is released .

Reply:
@snjeza  Perhaps you could use InitializeFinalFieldProposalCore  instead?  It's the core version of the class and it's available in jdt.core.manipulations.

Reply:
Trying...

Reply:
I have tried this issue using the jdt.core/jdt.ui master. It works.